### PR TITLE
Update Schedules.java

### PR DESCRIPTION
**Pull request by tenetiitma**

### CODE REVIEW

File: pratical_project/src/main/java/model/Schedules.java [hunk 1]
```diff
@@ -1,4 +1,32 @@
 package model;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
 public class Schedules {
```
Comment (ilja115610) [inline pratical_project/src/main/java/model/Schedules.java:9]: You could add @ Table (name = "schedules") as class level annotation

**Review by ilja115610 — APPROVED** [review]
Please see the comment

File: pratical_project/src/main/java/model/Schedules.java (hunk 1 above)
Comment (tenetiitma) [reply pratical_project/src/main/java/model/Schedules.java:7]: I have pushed the new one, is this correct?

Comment (ilja115610) [reply pratical_project/src/main/java/model/Schedules.java:7]: Yes

**Review by ilja115610 — APPROVED** [review]
Good

**Pull request merged**